compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -257,6 +311,17 @@ class ComposeScene internal constructor(
             mainOwner?.density = value
         }
 
+    /**
+     * The layout direction of the content, provided to the composition via [LocalLayoutDirection].
+     */
+    @ExperimentalComposeUiApi
+    var layoutDirection: LayoutDirection = layoutDirection
+        get() = mainOwner?.layoutDirection ?: field
```

Comment:
```suggestion
```
Better to remove the getter, so only ComposeScene will be the source of truth. Only the scene should change `mainOwner?.layoutDirection`

Reply:
On the other hand, I see this is how it's done for `density`...

Reply:
> we can't limit access to the owner

ComposeScene is intended to hide Owner as implementation detail. There was a plan to hide creation and controlling owners into the ComposeScene. So noone should control Owner except Scene by design.

Reply:
Done.

Reply:
We also need to remove getter here, so the scene is always return "field" value.

Reply:
I did, just haven't pushed until now.